packages/server/src/index.ts

```diff
@@ -13,6 +13,7 @@ export {
   type inferRouterOutputs,
   type AnyProcedure as AnyTRPCProcedure,
   type AnyRouter as AnyTRPCRouter,
+  type AnyMiddlewareFunction as AnyTRPCMiddlewareFunction,
```

Comment:
smurf all the things 
![CleanShot 2024-01-11 at 11 25 29](https://github.com/trpc/trpc/assets/459267/b830d6d3-2107-49d4-9eb8-93fcc90cdfb9)